File: src/com/geo/GeoQuake/WebInfoActivity.java
package com.geo.GeoQuake;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.webkit.WebView;


public class WebInfoActivity extends Activity {

    WebView mWebView;
    String mInfoUrl;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.web_info_activity_layout);

        mWebView = (WebView) findViewById(R.id.webView);
        mWebView.getSettings().setJavaScriptEnabled(true);
        mWebView.getSettings().setBuiltInZoomControls(true);
        Intent intent = getIntent();
        mInfoUrl = intent.getStringExtra("url");

        mWebView.loadUrl(mInfoUrl);

    }

    protected void onResume(){
        super.onResume();
    }

    protected void onDestroy(){
        super.onDestroy();
    }

}

File: src/com/geo/GeoQuake/WebInfoActivity.kt
package com.geo.GeoQuake

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View
import android.webkit.WebView


class WebInfoActivity : Activity() {

    lateinit var mWebView: WebView
    lateinit var mInfoUrl: String

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.web_info_activity_layout)

        mWebView = findViewById(R.id.webView)
        mWebView.settings.javaScriptEnabled = true
        mWebView.settings.builtInZoomControls = true
        val intent = intent
        mInfoUrl = intent.getStringExtra("url")

        mWebView.loadUrl(mInfoUrl)

    }

    override fun onResume() {
        super.onResume()
    }

    override fun onDestroy() {
        super.onDestroy()
    }

}